lib/wasix/src/runtime/package_loader/load_package_tree.rs

```diff
@@ -488,6 +488,12 @@ fn filesystem_v2(
             continue;
         }
 
+        if mount_path.as_path() == Path::new("/") {
+            bail!(
+                "The \"{package}\" package wants to mount a volume at \"/\", but that's not allowed",
+            );
+        }
```

Comment:
Maybe a helper function for this? It is used in fs v2 and v3. Something like (not tested),
```
fn check_volume_path(
    mount_path: &PathBuf,
    package: &str,
) -> Result<(), Error> {
    if mount_path.as_path() == Path::new("/") {
        bail!(
            "The \"{package}\" package wants to mount a volume at \"/\", but that's not allowed",
        );
    }
    Ok(())
}
```

Reply:
if only the code was that clean to begin with... it has been my experience that attempting to make clean implementations in code that's not clean by design usually leads to more complexity down the line, so when I see duplicate code, I also duplicate mine, to at least keep the code *consistently* unclean.